software-delivery-performance-terraform/atlassian.auto.tfvars

```diff
@@ -0,0 +1,62 @@
+# Sumo Logic - SDP Terraform
+# Configure Jira Cloud, Jira Server, OpsGenie, BitBucket credentials and parameters.
+
+
+# Jira Cloud
+jira_cloud_url      = "https://<EXAMPLE>.atlassian.net" # Please replace <EXAMPLE> with your Jira cloud instance id.
+jira_cloud_user     = "<YOUR JIRA CLOUD USERNAME>"      # Please replace <YOUR JIRA CLOUD USERNAME> (including brackets) with your Jira Cloud Username.
+jira_cloud_password = "<YOUR JIRA CLOUD PASSWORD>"      # # Please replace <YOUR JIRA CLOUD PASSWORD> (including brackets) with your Jira Cloud password or API Key, To generate API Key: https://confluence.atlassian.com/cloud/api-tokens-938839638.html
+jira_cloud_jql      = ""                                #jql = "project = Sumo" See, https://support.atlassian.com/jira-software-cloud/docs/what-is-advanced-searching-in-jira-cloud/
```

Comment:
Can add to comment: optional, value can be: "project = Sumo", see ...

Reply:
Done.